content/applications/inventory_and_mrp/manufacturing/workflows/allocation.rst

```diff
@@ -0,0 +1,142 @@
+==================
+Allocation reports
+==================
+
+.. |SO| replace:: :abbr:`SO (sales order)`
+.. |SOs| replace:: :abbr:`SOs (sales orders)`
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |MOs| replace:: :abbr:`MOs (manufacturing orders)`
+.. |RfQ| replace:: :abbr:`RfQ (request for quotation)`
+
+When fulfilling sales orders (SOs), or sourcing components for manufacturing orders (MOs), it is
+sometimes necessary to prioritize one |SO| or |MO| over another. In situations where there is
+insufficient stock on-hand to fulfill every |SO| or |MO|, ensuring that products and components are
+reserved for priority orders is essential.
+
+In Odoo *Manufacturing*, allocation reports are used on |MOs| to assign products to specific sales
+orders |SOs|, or components to specific |MOs|. This ensures the products or components are available
+for those orders, and are not used by mistake.
+
+Configuration
+=============
+
+To use allocation reports, the *Allocation Report for Manufacturing Orders* feature **must** be
+enabled. To do so, navigate to :menuselection:`Manufacturing app --> Configuration --> Settings`,
+and tick the checkbox next to :guilabel:`Allocation Report for Manufacturing Orders`. Then, click
+:guilabel:`Save`.
+
+.. image:: allocation/allocation-setting.png
+   :align: center
+   :alt: The Allocation Report for Manufacturing Orders setting.
```

Comment:
I dont think this image adds much value, since the instruction is very clear. Can remove :)
```suggestion
```